oidc-controller/api/authSessions/models.py

```diff
@@ -41,4 +42,5 @@ class AuthSessionCreate(AuthSessionBase):
 
 class AuthSessionPatch(AuthSessionBase):
     proof_status: AuthSessionState = Field(default=AuthSessionState.PENDING)
+    proof_expired_time: Union[datetime, None] = Field(default=None)
```

Comment:
Same as previous comment

Reply:
Same as above... Can consolidate if it won't break anything.